calico_containers/tests/st/test_duplicate_ips.py

```diff
@@ -0,0 +1,68 @@
+from sh import ErrorReturnCode_1
+from functools import partial
+
+from test_base import TestBase
+from docker_host import DockerHost
+
+
+class TestDuplicateIps(TestBase):
+    def test_duplicate_ips(self):
+        """
+        Start two workloads with the same IP on different hosts. Make sure they
+        can be reached from all places even after one of them is deleted.
+        """
+        host1 = DockerHost('host1')
+        host2 = DockerHost('host2')
+        host3 = DockerHost('host3')
+
+        calicoctl = "/code/dist/calicoctl %s"
+
+        host1.execute(calicoctl % ("node --ip=%s" % host1.ip))
```

Comment:
How about adding a `calicoctl()` method to the host?  Then you could do something like `host1.calicoctl("node", ip=host1.ip)` and get rid of all those string interpolations.